global.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
```

Comment:
I think the name - `global.go` doesn't represent the functionality. It should be `redirect.go`  or smth like that.